### PR TITLE
[STORM-1414] Some improvements for multilang JsonSerializer

### PR DESCRIPTION
**Pull request by vesense**
https://issues.apache.org/jira/browse/STORM-1414

### CODE REVIEW

**Comment by 2new** [comment]
+1 LGTM

File: storm-core/src/jvm/backtype/storm/multilang/JsonSerializer.java [hunk 1]
```diff
@@ -33,20 +33,24 @@
...
  */
 public class JsonSerializer implements ISerializer {
-    private DataOutputStream processIn;
+    //ANY CHANGE TO THIS CODE MUST BE SERIALIZABLE COMPATIBLE OR THERE WILL BE PROBLEMS
+    private static final long serialVersionUID = 8581179559440582569L;
+
+    public static final String DEFAULT_CHARSET = "UTF-8";
+
+    private BufferedWriter processIn;
     private BufferedReader processOut;
```
Comment (revans2) [inline storm-core/src/jvm/backtype/storm/multilang/JsonSerializer.java]: If we are setting the serialVersionUID (which I think is great) can we mark processIn and processOut as transient?

**Comment by revans2** [comment]
One very minor comment and then I am +1.

File: storm-core/src/jvm/backtype/storm/multilang/JsonSerializer.java (hunk 1 above)
Comment (vesense) [inline storm-core/src/jvm/backtype/storm/multilang/JsonSerializer.java]: Done.

**Comment by vesense** [comment]
comments addressed.

**Comment by revans2** [comment]
+1

**Pull request merged**